chapters/equations.tex

```diff
@@ -912,7 +912,7 @@ \subsection{Equations Needed for Initialization}\label{the-number-of-equations-n
 \end{lstlisting}
 
 Here we have three variables with unknown values: two dynamic variables that also are state variables, \lstinline!x1! and \lstinline!x2!, i.e., $n=2$, one output variable \lstinline!y!, i.e., $m=1$, and one input variable \lstinline!u! with known value.
-A consistent solution of the initial value problem providing initial values for \lstinline!x1!, \lstinline!x2!, \lstinline!der(x1)!, \lstinline!der(x2)!, and \lstinline!y! needs to be found.
+A consistent solution of the initialization problem requires initial values for \lstinline!x1!, \lstinline!x2!, \lstinline!der(x1)!, \lstinline!der(x2)!, and \lstinline!y! needs to be found.
```

Comment:
Can't have both _requires_ and _needs_:
```suggestion
A consistent solution of the initialization problem requires initial values for \lstinline!x1!, \lstinline!x2!, \lstinline!der(x1)!, \lstinline!der(x2)!, and \lstinline!y! to be found.
```

Reply:
Good catch, but thinking I believe it is even better to just write 'requires finding initial values for...'.

Reply:
Yes, here you go:
```suggestion
A consistent solution of the initialization problem requires finding initial values for \lstinline!x1!, \lstinline!x2!, \lstinline!der(x1)!, \lstinline!der(x2)!, and \lstinline!y!.
```

Reply:
OK, I just found your own suggestion in the comments below.  Resolving.